modules/openxr/openxr_api.cpp

```diff
@@ -1854,9 +1840,6 @@ OpenXRAPI::~OpenXRAPI() {
 	composition_layer_providers.clear();
 
 	// cleanup our extension wrappers
-	for (OpenXRExtensionWrapper *extension_wrapper : registered_extension_wrappers) {
-		memdelete(extension_wrapper);
-	}
```

Comment:
Do we have any means of cleaning the extension wrappers?

Reply:
This needs to move into `register_types.cpp/uninitialize_openxr_module` IMHO

Reply:
@konczg just a reminder, this would be nice to add into the `uninitialize_openxr_module` function

Reply:
This has been added with @BastiaanOlij 's changes: https://github.com/godotengine/godot/pull/68259/files#diff-29befd060761763bf49d5349defccbc36170ce85ce687d92743b312acc330cf4R191